xcube/core/normalize.py

```diff
@@ -201,12 +201,8 @@ def _normalize_lon_360(ds: xr.Dataset) -> xr.Dataset:
     new_vars = dict()
     for var_name in var_names:
         var = ds[var_name]
-        if len(var.dims) >= 1 and var.dims[-1] == 'lon':
-            values = np.copy(var.values)
-            temp = np.copy(values[..., : lon_size_05])
-            values[..., : lon_size_05] = values[..., lon_size_05:]
-            values[..., lon_size_05:] = temp
-            new_var = xr.DataArray(values, dims=var.dims, attrs=var.attrs)
+        if 'lon' in var.dims:
+            new_var = var.roll(lon=lon_size_05, roll_coords=False)
```

Comment:
Great! But why is `roll_coords=False`? Shouldn't they be rolled too?

Reply:
No, they shouldn't. We just took the first half of the lon coordinates and put them after the second half. The purpose of the roll command is that the variables are aligned with the correct longitude values again. If we would roll the coordinates, too, the relationship between the data and the dimension would not change and stay incorrect (see test test_fix_360_lon in test_normalize . It would fail if you'd set roll_coords to True).